DomainContent/LoadTest/VoteApp/voteApp.js

```diff
@@ -0,0 +1,411 @@
+"use strict";
+/* eslint-disable indent */
+
+//
+//  Example Vue App
+//
+//  Created by Milad Nazeri on 2018-10-11
+//  Modified from AppUi.js by Howard Stearns on 2 Nov 2016
+//  Copyright 2016 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+/* globals Tablet, Script, HMD, Controller, Menu */
+
+(function () { // BEGIN LOCAL_SCOPE
+    // Dependencies
+    // /////////////////////////////////////////////////////////////////////////
+    var AppUi = Script.require('./AppUi.js');
+    var Helper = Script.require('https://raw.githubusercontent.com/highfidelity/hifi-content/master/Utilities/Helper.js');
+
+    // Consts
+    // /////////////////////////////////////////////////////////////////////////
+    var URL = Script.resolvePath("./html/tablet.html"),
+        BUTTON_NAME = "VOTEAPP",
+
+        EXAMPLE_MESSAGE = "EXAMPLE_MESSAGE",
+        EVENT_BRIDGE_OPEN_MESSAGE = "eventBridgeOpen";
+
+    // DATA MANAGEMENT
+    var openPolls = {
+        avatar: false,
+        domain: true
+    };
+    var domainsInfo = {}; // will use Object.keys to get array
+    var avatarsInfo = {}; // array for now
+
+    var firstLoad = true;
+    var VOTE_APP_SETTINGS_NAME = "robinTest";
+    // JSON.stringify(Settings.setValue("robinTest", []));
+    // JSON.stringify(Settings.getValue("robinTest"));
+
+    // Init
+    // /////////////////////////////////////////////////////////////////////////
+    var ui;
+    var dataStore = {
+        openPolls: {
+            avatar: false,
+            domain: true
+        },
+        visitedAllDomains: false,
+        domains: [],
+        avatars: []
+    };
+    // Constructors
+    // /////////////////////////////////////////////////////////////////////////
+    // Collections
+    // /////////////////////////////////////////////////////////////////////////
+
+    // GOOGLE SHEET FUNCTIONS
+    function getGoogleData() {
```

Comment:
Seems like this isn't currently doing what you want it to be doing, is that right?

Reply:
There's a lot of commented-out code in this file, which leads me to believe it isn't quite ready for CR yet. I'm going to stop CRing this file for now. When you reach this comment, please let me know this file's status before I continue :)